app/models/piece.rb

```diff
@@ -0,0 +1,7 @@
+class Piece < ActiveRecord::Base
```

Comment:
Take a look at scopes - http://katieleonard.ca/blog/2015/scopes-and-single-table-inheritance/